### PR TITLE
Added Github Actions for testing PRs for Docker Builds (Issue #83)

### PR DESCRIPTION
**Pull request by Sak1012**
This pull request introduces a new GitHub Actions workflow (docker-pr.yml) to automate the building of Docker images for the eventyay-tickets project. The workflow is triggered whenever a pull request is opened or updated targeting the development or main branches. Additionally, it checks and installs Python dependencies before building the Docker image.

- Tries to solve issue #83 

### Changes Made
- Added a new workflow file docker-pr.yml in the .github/workflows directory.
- Configured the workflow to check and install Python dependencies using the requirements.txt file located in the src directory.
- Built the Docker image using the installed dependencies.
- Ensured that the workflow only runs for pull requests targeting the development or main branches.
- Tested the workflow locally to ensure proper functionality.

Test Getting triggered on a pull-request made to the development branch of my local repository.
Screenshot 2024-05-15 at 00 34 17

Success message of the test while trying to create a PR in-between to local branches aimed at development branch
Screenshot 2024-05-14 at 15 28 56

### CODE REVIEW

**Comment by marcoag** [comment]
@Sak1012 Thanks a lot for the PR, I had a look at commit https://github.com/fossasia/eventyay-tickets/pull/85/commits/19975bb625edbb73e6aa88c46221f2e5fb264106 and it looks pretty good. Unfortunately we had to run a hard reset on the development branch due to a messed up merge, would you mind cherry picking the relevant changes and opening a PR against the current development branch? 

If you have trouble doing it or you prefer I can do it myself, I just didn't want to steal the authorship of this PR from you ;)

**Review by marcoag — CHANGES_REQUESTED** [review]
Please open a new PR against current `development` branch.

**Review by marcoag — APPROVED** [review]

**Review by norbusan — APPROVED** [review]
LGTM, we will see in reality how well it fares.

I would probably fix a certain Python version, the one that is required by ticekts.

File: .github/workflows/docker-pr.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
```
Comment (norbusan) [inline .github/workflows/docker-pr.yml]: Can we use 3.8 here as we do with all other versions and tests?

Comment (Sak1012) [reply .github/workflows/docker-pr.yml]: I have updated the Version to `"3.8"` and the build is successful, please take a look

image

**Comment by mariobehling** [comment]
Ok, I am merging this to test.

**Pull request merged**